modules/ensemble/lib/ensemble.dart

```diff
@@ -361,6 +362,24 @@ class Ensemble extends WithEnsemble with EnsembleRouteObserver {
     return null;
   }
 
+  Object? getSelectedLanguage() {
+    UserLocale? userLocale = UserLocale.from(Ensemble().getLocale());
+    String languageCode = userLocale!.languageCode;
```

Comment:
userLocale can be null, you know that as you put it as optional with a ?. How can you then turn around and put the ! to force it to be non-null! This is basic stuff! check the rest of your code for such basic stuff before you post the PR again!